controllers/index.js

```diff
@@ -51,9 +56,15 @@ exports.index = (req, res) => {
         }).catch((err) => {
             console.log(err);
             if (req.xhr) {
-                res.status(404).render('partials/error', {layout: false});
+                if (err === 'notUser') {
+                    res.status(400).render('partials/error-user', {layout: false});
+                } else {
+                    res.status(404).render('partials/error', {layout: false});
+                }
             } else {
-                res.render('index', {error: true, username});
+                res.render('index', {
+                  error: true, errorUser: err === 'notUser', username
```

Comment:
How about always passing the message in error rather than boolean then we can just use the existing error stuff and don't need all the conditions around errorUser and an additional partial?

Reply:
Good call. Done. :+1: